recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "unixODBC" %}
-{% set version = "2.3.9" %}
-{% set sha256 = "52833eac3d681c8b0c9a5a65f2ebd745b3a964f208fc748f977e44015a31b207" %}
+{% set version = "2.3.11" %}
+{% set sha256 = "d9e55c8e7118347e3c66c87338856dad1516b490fb7c756c1562a2c267c73b5c" %}
 
 package:
   name: {{ name|lower }}
```

Comment:
Could you please fix the license name as `LGPL-2.1-or-later` (the license file is inside http://www.unixodbc.org/unixODBC-2.3.11.tar.gz).
The comment:
```
    # patches:
    #   - 0001-Add-support-for-editline.patch
```
can be removed.